types/user.ts

```diff
@@ -586,6 +718,14 @@ export class User extends LazyObject {
 
   // ======================== Trivial Getters Below ========================
 
+  /** getAuthor
+   * @returns this user :)
+   * @remarks this is here to make User work cleanly with reporting functions, avoids ternary operators
+   */
+  public getAuthor() {
+    return this;
```

Comment:
the fuck?

Reply:
most industry-level code I've ever written tbh

Reply:
No but like why is this code here? like, if i understand correctly... `this.getAuthor()` is equivalent to just saying `this`

Reply:
look at the code that references it. As the comment says, it makes the reporting functions cleaner imo